packages/components/utils/src/getStringMatch/getStringMatch.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: 'getStringMatch'
+type: 'component'
+status: 'beta'
+slug: /utils/get-string-match/
+typescript: ./getStringMatch.ts
+---
+
+import { Props } from '@contentful/f36-docs-utils';
+
+Function that can be used to find a substring inside another string.
+
+## Table of contents
+
+- [Import](#import)
+- [Examples](#examples)
+  - [Basic usage](#basic-usage)
+- [Props](#props)
+- [Help improve this page](#help-improve-this-page)
+
+## Import
+
+```jsx static=true
+import { getStringMatch } from '@contentful/f36-utils';
+```
+
+## Examples
+
+### Basic usage
+
+You pass the base string as a first argument, and a string to match it with as a second argument. The function will return an object containing the match, what comes before the match, and what comes after the match in the base string.
+
+```jsx static=true
```

Comment:
@thebesson I am not sure if that's intended here, but if you would like to display examples, they can not have `static=true`. Or you add this because they were broken and it was the only way to show them?

Reply:
yes, without `static=true` and without imports there was an error. and imports cannot be used without `static=true`. That's the problem

Reply:
hmmm, and is the `@contentful/f36-utils` installed? maybe it's missing in the dependencies?

Reply:
it's inside the `@contentful/f36-utils`, so I don't know

Reply:
You will need to pass this to the scope prop of LiveProvider
https://github.com/contentful/forma-36/blob/c08926fd6734339eab43fe98af5fbd947495d7b9/packages/forma-36-website/src/components/ComponentSource.js#L51-L57

as you can see we are already passing our components, icons, and tokens. If you remove icons, for example, you will see that some pages will break and that happens because react-live tries to render your code but does not have the dependencies it needs

Reply:
makes sense 👍